### DISCUSSION TITLE
View - Component State Management Strategy

### DISCUSSION THREAD

**Discussion by jin-Pro** [githru/githru-vscode-ext#69] [View]
> View 각 Feature에서 컴포넌트 렌더링 조건을 확립하고자 해당 Discussion을 작성하였습니다.

## [ Temporal Filter 에서 구간 설정  ]
- 설정 구간 데이터 VerticalClusterList Render
- 설정 구간 데이터 Statistics Render

## [ VerticalClusterList 에서 특정 Cluster Click ]
- Statistics Component에서 해당 Cluster Data Render
- Vertical Cluster List 에서 해당 Cluster 에 존재하는 Commit 내역 Render

## [ Vertical Cluster List 에서 특정 Commit Click ]
- Statistics Component에서 해당 Commit Data Render
- Detail Component에서 해당 Commit Data Render

---

위 흐름으로 이해하고있는데 검토 부탁드립니다!

**Comment by jejecrunch** [comment]
여기에 따로 작성하지 않았네요 ㅜㅜ

VerticalClusterList에서 특정 Cluster Click 이벤트와 특정 Commit Click 이벤트는 동일하게 Detail 컴포넌트를 보여줍니다.
이때 Detail 컴포넌트는 한 클러스터 안에 있는 커밋 전체 목록을 보여줍니다.

위 사항은 디스코드로 전달된 사항이지만 discussion에 따로 답변도 드려야 할 것 같아 드립니다 !
